src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
please add empty line between siblings. check all other cases
```suggestion
          </li>
          
          <li class="nav__item">
```